magento/magento2
# #13157 - Last Ordered Items block - bad js code

### Description
Fix logic in last-order-items.js and Recently Ordered block.

### Fixed Issues
magento/magento2#13157: Last Ordered Items block - bad js code

### Manual testing scenarios
1. Create order for registered customer
2. Verify that Add to Cart button in Recently Ordered block reflects is_salable property of recently ordered product

### Contribution checklist
